jwst/transforms/models.py

```diff
@@ -1491,7 +1491,7 @@ def evaluate(self, x, y, x0, y0, order):
         t = np.linspace(0, 1, 10)  #sample t
         dx = self.xmodels[iorder][0](x0, y0) + t * self.xmodels[iorder][1](x0, y0)
         dy = self.ymodels[iorder][0](x0, y0) + t * self.ymodels[iorder][1](x0, y0)
-        if self.theta != 0.0:
+        if self.theta > 0.0:
```

Comment:
hmm...theta here is the niriss filter wheel offset, taken from header keywords. It's needed in a couple models, but I'm thinking it should go back to !=0 in all, it may be possible for the FWCPOS in the obs to be greater than the reference position and we still want to rotate the traces. This might have been part of the merge issue that I missed :/